product-matrix.json

```diff
@@ -50,18 +50,18 @@
     },
     {
       "channel": "stable",
-      "comments": "IntelliJ 2025.1 Beta, Android Studio Narwhal 2025.1 Canary",
+      "comments": "IntelliJ 2025.1, Android Studio Narwhal 2025.1",
       "name": "2025.1",
       "version": "2025.1",
       "ijVersion": "2025.1",
-      "ideaProduct": "IC",
-      "ideaVersion": "251.23774.16",
+      "ideaProduct": "android-studio",
+      "ideaVersion": "2025.1.1.6",
       "baseVersion": "251.23774.16",
       "dartPluginVersion": "251.23774.16",
-      "androidPluginVersion": "251.23774.16",
-      "sinceBuild": "243",
+      "androidPluginVersion": "",
+      "sinceBuild": "251",
       "untilBuild": "253.*",
-      "isUnitTestTarget": "true",
+      "isUnitTestTarget": "false",
```

Comment:
I'm trying to understand what this is doing and it seems like it serves a few purposes... Maybe you can explain in person?